src/tests/tests.spec.js

```diff
@@ -0,0 +1,108 @@
+import 'dotenv/config';
+
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.describe('filtering checks', () => {
+    test.beforeEach(async ({ loginPage }) => {
+        const userName = process.env.STANDARD_USER_NAME;
+        const password = process.env.PASSWORD;
```

Comment:
I would move this logic to some userDataHelper.util.js so every time you need to get it, you will have a single place with it. Otherwize once you switch from STANDARD_USER_NAME to USER_MYKOLA_NAME, you will need to refactor all the places using this login info.